extension/src/background/helpers/dataStorage.ts

```diff
@@ -254,6 +256,18 @@ export const resetAccountSubscriptions = async () => {
     // once account is unlocked, setup Mercury account subscription if !HAS_ACCOUNT_SUBSCRIPTION
     await localStore.setItem(HAS_ACCOUNT_SUBSCRIPTION, {});
   }
+  await migrateDataStorageVersion("4.0.2");
```

Comment:
the `resetAccountSubscription` migration forgot to migrate the version, so adding this in here now

Reply:
omg, thank you